src/pynear/account.py

```diff
@@ -370,15 +370,21 @@ async def view_function(
         result["result"] = json.loads("".join([chr(x) for x in result["result"]]))
         return ViewFunctionResult(**result)
 
-    async def get_balance(self, account_id: str = None) -> int:
+    async def get_balance(self, account_id: str = None, token_id : str = None) -> int:
```

Comment:
There is `account.fts.get_balance()` for this
I don't think it's necessary to duplicate functionality

Reply:
ok, got it